.github/workflows/build.yml

```diff
@@ -58,16 +59,22 @@ jobs:
           sudo apt-get install -y qemu-system-arm binfmt-support qemu-user-static
           docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
 
-          echo "Running pyinstaller"          
+          echo "Running pyinstaller"
+          PYTHON_VERSION=$(echo ${PYTHON_VER} | cut -d. -f1,2)
+          echo "Using Python version: ${PYTHON_VERSION}"
+          
           docker run --rm --name builder \
             --platform=aarch64 \
             --user=$(id -u):$(id -g) \
             -v ${GITHUB_WORKSPACE}:/workspace \
+            -e PYTHON_VERSION=${PYTHON_VERSION} \
             docker.io/unfoldedcircle/r2-pyinstaller:${PYTHON_VER} \
             bash -c \
               "cd /workspace && \
-              python -m pip install -r requirements.txt && \
-              pyinstaller --clean --onedir --name intg-denonavr --add-data intg-denonavr/locales:locales intg-denonavr/driver.py"
+              pip install --user -r requirements.txt && \
+              pip install --user ./denonavrlib && \
+              pip install --user --force-reinstall pyinstaller && \
```

Comment:
The newly added `--force-reinstall pyinstaller` option will potentially install a new version if available, which could break a build.

The intention of the versioned Docker build image is to have a pinned pyinstaller version for a defined build state. A new pyinstaller version requires a new build image version.

Please remove.